index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS only-->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <title>microverse survey</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <h1 id="title">Microverse collaboration survey form</h1>
+      <p id="description">
+        This is a survey for the Microverse international applicants who are
+        working on various projects in our Microverse trials
+      </p>
+
```

Comment:
I think it would be better if you used semantic tags in your project i.e. headers

Reply:
- [OPTIONAL] Please, try to explain more the reasoning for your suggestion ( what are the advantages of semantic tags? )

> for example, you can add 
> I think it would be better if you used semantic tags in your project i.e. headers. Using semantic tags correctly make it clear to the browser what the meaning of a page and its content is. That clarity is also communicated with search 👍 engines, ensuring that the right pages are delivered for the right queries.